lib/config/presets/internal/replacements.ts

```diff
@@ -743,6 +744,29 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'opencost-registry-move': {
+    description: 'Replace OpenCost registry from quay.io to ghcr.io.',
+    packageRules: [
+      {
+        description:
+          'Replace `quay.io/kubecost1/kubecost-cost-model` with `ghcr.io/opencost/opencost`.',
+        matchCurrentVersion: '1.108.0',
```

Comment:
is it the last version published there?

Reply:
Correct.

That's the version Renovate will provide when using quay.io: https://github.com/MindTooth/renovate-opencost-test/pull/1

Reply:
And were 109 and 110 skipped?

Reply:
They moved over to ghcr.io.  Hence this replacement.

1.110.0 was wanted by the maintainer, but you maybe want the most lowest version first (1.109.0)?

Reply:
ok, use lowest new version as replacement

Reply:
Done.